CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - On Wayland, implement `request_user_attention` with `xdg_activation_v1`.
 - On X11, emit missing `WindowEvent::ScaleFactorChanged` when the only monitor gets reconnected.
 - On X11, if RANDR based scale factor is higher than 20 reset it to 1
+- On Wayland, feature `wayland-dlopen` to use `dlopen` for opening system libraries enabled by default.
```

Comment:
```suggestion
- On Wayland, add an enabled-by-default feature called `wayland-dlopen` so users can opt out of using `dlopen` to load system libraries.
```
The important change here is that users can opt out, so I think we should call that out explicitly.